partitura/utils/music.py

```diff
@@ -319,6 +321,18 @@ class MIDITokenizer(object):
 # Standard tuning frequency of A4 in Hz
 A4 = 440.0
 
+COMPOSITE_DURS = np.array(
```

Comment:
this should be a dictionary.
And we put a comment on top saying that this is a shortcut to make it work in common cases, but a more complex algorithm is necessary to handle general cases.

Reply:
Generally I agree, I just did it identical with the way it was done before for other durations for consistency.